### PR TITLE
A bit more info about _vshIoMount/vshIoUmount

### PR DESCRIPTION
**Pull request by v-atamanenko**

### CODE REVIEW

**Comment by v-atamanenko** [comment]
@d3m3vilurr can you merge?

**Comment by v-atamanenko** [comment]
@Princess-of-Sleeping can you merge? :)

File: include/psp2/vshbridge.h [hunk 1]
```diff
@@ -43,25 +65,32 @@ int _vshSblAimgrGetConsoleId(char CID[32]);
 SceUID _vshKernelSearchModuleByName(const char *module_name, const void *buffer);
 
 /**
- * @param[in] id - mount id
- * @param[in] path - mount path
- * @param[in] permission - 1/RO 2/RW
- * @param[in] buf - work buffer
+ * @brief Mount a partition.
+ *
+ * @param[in] id - Mount ID.
```
Comment (Rinnegatamante) [inline include/psp2/vshbridge.h:70]: You could add here `(One of ::SceVshMountId)`

File: include/psp2/vshbridge.h [hunk 2]
```diff
@@ -43,25 +65,32 @@ int _vshSblAimgrGetConsoleId(char CID[32]);
...
- * @param[in] force - Set to 1 to force umount
- * @param[in] unk2 - Unknown, set 0
- * @param[in] unk3 - Unknown, set 0
+ * @brief Unmount a partition.
+ *
+ * @warning Unmounting system partitions can cause system instability, even if
+ *          you remount them back. For example, FIOS overlays used by the Shell
+ *          won't get recreated and would effectively render the Shell unusable
+ *
+ * @param[in] id - Mount ID.
```
Comment (Rinnegatamante) [inline include/psp2/vshbridge.h:86]: same here

**Review by Rinnegatamante — CHANGES_REQUESTED** [review]

**Pull request merged**